app/src/main/java/id/rizmaulana/covid19/ui/overview/DashboardViewModel.kt

```diff
@@ -27,47 +33,62 @@ class DashboardViewModel(
     val errorMessage: LiveData<String>
         get() = _errorMessage
 
-    private val _dailyListData = MutableLiveData<List<CovidDaily>>()
-    val dailyListData: LiveData<List<CovidDaily>>
-        get() = _dailyListData
+    private val _liveItems = MutableLiveData<List<BaseViewItem>>()
+    val items: LiveData<List<BaseViewItem>>
+        get() = _liveItems
 
-    private val _overviewData = MutableLiveData<CovidOverview>()
-    val overviewData: LiveData<CovidOverview>
-        get() = _overviewData
+    fun loadOverviewAndDaily() {
+        val overviewObservable = appRepository.overview()
+            .subscribeOn(schedulerProvider.io())
```

Comment:
you can remove this one, it is already subscribed on IO thread from `networkModule`:
https://github.com/rizmaulana/kotlin-mvvm-covid19/blob/master/app/src/main/java/id/rizmaulana/covid19/di/NetworkModule.kt

```kt
...
.addCallAdapterFactory(RxJava2CallAdapterFactory.createWithScheduler(Schedulers.io()))
...
```

Reply:
Okayy 👍